recipes-debian/libnss-mdns/libnss-mdns_debian.bb

```diff
@@ -0,0 +1,47 @@
+# base recipe: meta/recipes-support/libnss-mdns/libnss-mdns_0.10.bb
```

Comment:
The libnss-mdns is in recipes-connectivity, isn't it?

Reply:
Ouch!